include/cutlass/epilogue/collective/builders/pvc_builder.inl

```diff
@@ -0,0 +1,130 @@
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
+ *
+ * 3. Neither the name of the copyright holder nor the names of its
+ * contributors may be used to endorse or promote products derived from
+ * this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ **************************************************************************************************/
+
+#pragma once
+
+#include <cutlass/arch/arch.h>
+
+#include "cutlass/epilogue/collective/default_epilogue.hpp"
+#include "cutlass/epilogue/collective/intel_pvc_epilogue.hpp"
+#include "cutlass/epilogue/fusion/intel_pvc_callbacks.hpp"
+
+
+namespace cutlass::epilogue::collective {
+
+  // PVC epilogue builder
+
+template <
+  class TileShape_MNK,
+  class EpilogueTileType,
+  class ElementAccumulator,
+  class ElementCompute,
+  class ElementC_,
+  class GmemLayoutTagC_,
+  int AlignmentC,
+  class ElementD,
+  class GmemLayoutTagD,
+  int AlignmentD,
+  class FusionOpOrCallbacks
+  >
+  class CollectiveBuilder<
+      arch::IntelPVC,
+      arch::OpClassTensorOp, 
+      TileShape_MNK,
+      Shape<_1, _1, _1>,    // Cluster Shape, which has no meaning in PVC
+      EpilogueTileType,
+      ElementAccumulator,
+      ElementCompute,
+      ElementC,
+      GmemLayoutTagC,
+      AlignmentC,
+      ElementD,
+      GmemLayoutTagD,
+      AlignmentD,
+      EpilogueScheduleAuto, // We do not have different type of epilogue support yet
+      FusionOpOrCallbacks,
+      cute::enable_if_t<
+        cute::is_same_v<GmemLayoutTagC,  cutlass::layout::RowMajor> &&
+        cute::is_same_v<GmemLayoutTagD,  cutlass::layout::RowMajor> &&
+        cute::is_same_v<EpilogueTileType, EpilogueTileAuto> &&
+        // Only linear combination is supported at the moment
+        cute::is_same_v<FusionOpOrCallbacks, 
+                        cutlass::epilogue::fusion::LinearCombination<ElementD,ElementCompute,ElementC,ElementCompute>>
+      >
+    >{
+      #ifndef SYCL_INTEL_TARGET
+        static_assert(cutlass::detail::dependent_false<arch::IntelPVC>, 
```

Comment:
Same for this `static_assert` as well.

Reply:
This has now been changed to check if target is Nvidia rather than particularly checking for PVC